src/commands/create-scene.ts

```diff
@@ -241,72 +127,91 @@ f79d4760-0b71-44e4-ad0b-22743fdd4ca3
                 suppliedId: i.suppliedId,
                 transform: i.transform,
                 visible: true,
+                name: i.name ?? i.suppliedId ?? undefined,
+                ordinal: i.ordinal ?? undefined,
               },
               relationships: {},
               type: 'scene-item',
             },
-          })
-        );
+          });
+        }
+      });
 
-        if (showProgress) progress.start(createSceneItemReqs.length, 0);
+      const res = await createSceneFn({
+        client,
+        createSceneItemReqs: createSceneItemReqs,
+        createSceneReq: () => ({
+          data: {
+            attributes: { name, suppliedId, treeEnabled },
+            type: SceneRelationshipDataTypeEnum.Scene,
+          },
+        }),
+        failFast: !noFailFast,
+        onMsg: console.error,
+        onProgress: (complete, total) => {
+          if (showProgress) progress.update(complete);
```

Comment:
nit: Just a preference thing here, but could we include all if statement code in brackets?
```ts
if (showProgress) {
   progress.update(complete);
}
```